pylti/flask.py

```diff
@@ -96,15 +96,32 @@ def verify(self):
 
     def _verify_any(self):
         """
-        Verify that request is in session or initial request
+        Verify that an initial request has been made, or failing that, that
+        the request is in the session
 
         :raises: LTIException
         """
         log.debug('verify_any enter')
-        try:
-            self._verify_session()
-        except LTINotInSessionException:
+
+        # Check to see if there is a new LTI launch request incoming
+        newrequest = False
+        if flask_request.method == 'POST':
+            params = flask_request.form.to_dict()
```

Comment:
update test `test_access_to_oauth_resource_without_authorization_any` because now it is verifying session `self._verify_session()` because `lti_message_type=basic-lti-launch-request` is not supplied